sys-cluster/easybuild/easybuild-4.6.2.ebuild

```diff
@@ -0,0 +1,35 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..11} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1
+
+DESCRIPTION="EasyBuild is a software build and installation framework."
+HOMEPAGE="
+	https://easybuild.io/
+	https://github.com/easybuilders/easybuild
+"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+lmod modules"
+REQUIRED_USE=" ^^ ( lmod modules ) "
+
+RDEPEND="
+	dev-libs/openssl
+	dev-tcltk/tclx
+	lmod? ( >=sys-cluster/lmod-6.5.1 )
+	modules? ( >=sys-cluster/modules-3.2.10 )
```

Comment:
If these are optional runtime dependencies only, perhaps we can use `optfeature` instead?

Reply:
They are not optional, since easybuild needs one module tool -> `REQUIRED_USE` (iirc modules and lmod can not both be installed).
https://docs.easybuild.io/en/latest/Installation.html#requirements

Reply:
Why can't we install both at the same time? If there is a problem with this then we shouldn't put this condition here, but add it to the `sys-cluster/modules` ebuild as a runtime blocker instead. 

I think here we want a `RDEPEND="|| ( sys-cluster/lmod sys-cluster/modules )"`

Reply:
I agree, they use the same named binary `module` (I guess to make switching the module tool easier):

* https://lmod.readthedocs.io/en/latest/045_transition.html#purge-modules-using-old-module-command
*  https://modules.sourceforge.net/#Quick_examples

Reply:
I created a revbump for modules with lmod strong blocked https://github.com/gentoo/sci/pull/1189

Then I'd adjust this PR'S modules version to include the revbump.

Reply:
> I created a revbump for modules with lmod strong blocked #1189.
> 
> Then I'd adjust this PR's modules version to include the revbump.

Great, Thanks

I still think we don't need the USE flags to select either lmod or modules, an `|| ( )` runtime dependency should be sufficient since we can switch between these two without having to recompile the package, correct?

Reply:
Yes, that makes more sense. I guess the first dependency in `|| ( )` is preferred?

Reply:
> Yes, that makes more sense. I guess the first dependency in `|| ( )` is preferred?

Yes, it is.

Reply:
Done.